eclipse-scout-core/src/dataobject/serialize/DataObjectDeserializer.ts

```diff
@@ -49,6 +49,16 @@ export class DataObjectDeserializer implements DataObjectDeserializerModel, Obje
     return resultObj;
   }
 
+  protected _resolveDataObjectType(rawObj: Record<string, any>, metaData: DoValueMetaData): Constructor {
```

Comment:
metaData is optional in _deserializeObject. Therefore it should be optional in _resolveDataObjectType and in DoTypeResolver.resolve as well?